src/Gui/SoFCCSysDragger.cpp

```diff
@@ -808,7 +809,7 @@ SoGroup* RDragger::buildGeometry()
 
     unsigned int segments = 15;
 
-    float angleIncrement = static_cast<float>(M_PI / 2.0) / static_cast<float>(segments);
+    float angleIncrement = (std::numbers::pi_v<float> / 2.f) / static_cast<float>(segments);
```

Comment:
We standardized on using capital `F` I think.

Reply:
I'm so used to using lower case that it takes a while to change. Thanks for reminding me.
In this file however, the style is to use lower case, should I still change this to upper?

Reply:
Doesn't really matter, we will eventually apply an autoformatter anyway. Just caught my eye because it is a thing I'm working to remember myself 😎.